DWM/update_abos_attributes_to_dwm.py

```diff
@@ -1,51 +1,43 @@
 #!/usr/bin/env python3
 """Rename ABOS files and attributes"""
+# As part of the ABOS renaming Odyssey, this script identifies the attributes that contains the words "ABOS" and
+# "Bluewater" (as art of the Australian Bluewater Observation System), changes them for "DWM"/"Deep Water Moorings"
+# and rename the files. The attribute 'history' is excluded from the search as we decide that attribute did not need
+# to be changed.
+
 
 import warnings
 import os
 import netCDF4 as nc
+import sys
 from typing import List
 from glob import glob
 
 
-# def test_replace_abos_str():
-#     string = 'my_test_XXX'
-#     old = 'XXX'
-#     new = 'YYY'
-#     expected = 'my_test_YYY'
-#     result = replace_abos_str(string, old, new)
-#     assert(result == expected)
-
+# search in the attributes which one has 'ABOS' in it, excluding the 'history' attribute (we do not need to change this)
 def find_att_with_str(ncobj: nc.Dataset, string: str = 'ABOS'):
```

Comment:
Instead of comments above each function, these descriptions can also become [docstrings](https://www.datacamp.com/community/tutorials/docstrings-python) for each function, e.g.
```python
def find_att_with_str(ncobj: nc.Dataset, string: str):
    """Search in the attributes which one has 'ABOS' in it, excluding the 'history' attribute """
```